pkg/api/validation/events_test.go

```diff
@@ -49,6 +49,28 @@ func TestValidateEvent(t *testing.T) {
 				},
 			},
 			false,
+		}, {
+			&api.Event{
+				ObjectMeta: api.ObjectMeta{
+					Name:      "test3",
+					Namespace: "default",
+				},
+				InvolvedObject: api.ObjectReference{
+					Kind: "Node",
+				},
+			},
+			true,
+		}, {
+			&api.Event{
+				ObjectMeta: api.ObjectMeta{
+					Name:      "test4",
+					Namespace: "default",
+				},
+				InvolvedObject: api.ObjectReference{
+					Kind: "Namespace",
+				},
+			},
+			true,
```

Comment:
Maybe add a test for an event about a job or something else in the extensions group.